baker/siteRenderers.tsx

```diff
@@ -452,6 +444,63 @@ ${posts
     return feed
 }
 
+export async function makeAtomFeedFromDataInsights({
+    dataInsights,
+    htmlUrl = BAKED_BASE_URL,
+    feedUrl = `${BAKED_BASE_URL}/atom.xml`,
+}: {
+    dataInsights: GdocDataInsight[]
+    htmlUrl?: string
+    feedUrl?: string
+}) {
+    return `<?xml version="1.0" encoding="utf-8"?>
+<feed xmlns="http://www.w3.org/2005/Atom">
+<title>"Our World in Data - Data Insights"</title>
```

Comment:
I don't think this should have quote marks, and I think it should say **Daily** Data Insights  🙂